dawn/examples/python/run.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
```

Comment:
We need to update the Jenkins tests to use `generate_and_diff` instead of this. See `Dockerfile` for specific command.